client/src/component/Milestone.js

```diff
@@ -0,0 +1,28 @@
+import "./Home.js";
+import React, { useEffect, useState } from "react";
+
+const ModuleInformation = () => {
+  const [module] = useState({
+    name: "Final Project",
+    deadline: new Date("2023-07-08"),
+  });
```

Comment:
There is generally no need to use useState when you are not using the set function, other than very rare occasions. This could just be an object stored outside this component but happy to leave it here for now as we will replace it when we can call the data from the database.